sdv/constraints/tabular.py

```diff
@@ -206,6 +213,15 @@ class FixedCombinations(Constraint):
     _combinations_to_uuids = None
     _uuids_to_combinations = None
 
+    @classmethod
+    def _validate_inputs(cls, **kwargs):
+        errors = super()._validate_inputs(
+            valid_parameters={'column_names'},
+            constraint='a FixedCombinations',
+            **kwargs
+        )
+        raise MultipleConstraintsErrors('\n' + '\n\n'.join(map(str, errors)))
```

Comment:
each of these just passes the parameters from the init right? And the constraint parameter is always 'a' + the name of the constraint? If so, I think all of this can just be moved into the base. We can use [introspection ](https://stackoverflow.com/questions/36849837/introspecting-arguments-from-the-constructor-function-init-in-python) to get the names of the required parameters for the class, and just access the name directly (ie. `class.__name__`). I don't think each class needs to override this method

Reply:
Sure, I can go for it. I'll have to add an if statement in the base to catch the`**an** Inequality` article, but that's minor. I'll also have to make the base raise the error, instead of returning a list of strings, so the two classes that do overwrite `_validate_inputs` will have to try/catch the error.

Reply:
Yeah, or the base method can just gather the strings still, and another method can call it and raise the `MultipleConstraintsError`